gapic/src/main/com/google/gapid/util/Logging.java

```diff
@@ -165,44 +183,25 @@ private static void protosToString(LogRecord rec) {
       }
       rec.setParameters(params);
     }
+  }
 
-    private static final String JAVA_PREFIX = "java.";
-    private static final String GAPID_PREFIX = "com.google.gapid.";
-    private static final String GOOG_PREFIX = "com.google.";
-
-    private static String getClassName(String className) {
-      if (className.startsWith(JAVA_PREFIX)) {
-        return "j." + className.substring(JAVA_PREFIX.length());
-      } else if (className.startsWith(GAPID_PREFIX)) {
-        return className.substring(GAPID_PREFIX.length());
-      } else if (className.startsWith(GOOG_PREFIX)) {
-        return "cg." + className.substring(GOOG_PREFIX.length());
-      }
-      return className;
-    }
-
-    private static char getLogLevel(int level) {
-      switch (level) {
-        case 1000: return 'E';
-        case  900: return 'W';
-        case  800: case  700: return 'I';
-        case  500: case  400: case  300: return 'D';
-        default:
-          if (level > 1000) {
-            return 'E';
-          } else if (level > 900) {
-            return 'W';
-          } else if (level > 700) {
-            return 'I';
-          } else {
-            return 'D';
-          }
-      }
+  private static final String JAVA_PREFIX = "java.";
+  private static final String GAPID_PREFIX = "com.google.gapid.";
+  private static final String GOOG_PREFIX = "com.google.";
+
+  private static String shorten(String className) {
```

Comment:
The JVM has no concept (almost) of inner classes. To the JVM, an inner class and its container outer class are simply two different classes. You cannot call private methods of another class. The compiler adds synthetic methods to make it look like you can call private methods from inner/outer classes. These synthetic methods add unnecessary overhead and pollute stack traces. If you're going to allow access via these synthetic methods, why not give access directly?